config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -152,6 +152,32 @@ presubmits:
           requests:
             memory: "6Gi"
 
+  - name: pull-kubernetes-e2e-scheduler-performance
+    always_run: true
+    max_concurrency: 12
+    decorate: true
+    annotations:
+      testgrid-create-test-group: "true"
+    branches:
+    - master
```

Comment:
Does this mean we download the master branch? We  should run tests for the PR that contains the code change.

Reply:
We always apply the PR - the question is for PRs from which branch we run tests.

Reply:
Thanks @liu-cong @wojtek-t 
To begin with, I think we want to test the PRs against master branch. 
Also, our [periodic job](https://github.com/kubernetes/test-infra/blob/7acdd045f98d72e44120eaa053d594bcd273c0ba/config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml#L358) is based on the master branch, I don't know if it will work properly on the other branches.
WDYT?

Reply:
Sorry my misunderstanding. I thought this means we run test against the master branch without applying the change.

This LGTM.